component/component/slos.libsonnet

```
@@ -88,8 +88,8 @@ local prometheusRule(name) =
         sli: {
           events: {
             // The  0*rate(...) makes sure that the query reports an error rate for all instances, even if that instance has never produced a single error
-            error_query: '(sum(rate(appcat_probes_seconds_count{reason!="success", service="VSHNPostgreSQL", ha="true"}[{{.window}}]) or 0*rate(appcat_probes_seconds_count{service="VSHNPostgreSQL"}[{{.window}}])) by (service, namespace, name, organization, sla) or vector(0)) > 0',
-            total_query: '(sum(rate(appcat_probes_seconds_count{service="VSHNPostgreSQL",  ha="true"}[{{.window}}])) by (service, namespace, name, organization, sla) or vector(0)) > 0',
+            error_query: 'sum(rate(appcat_probes_seconds_count{reason!="success", service="VSHNPostgreSQL", ha="true"}[{{.window}}]) or 0*rate(appcat_probes_seconds_count{service="VSHNPostgreSQL"}[{{.window}}])) by (service, namespace, name, organization, sla) or vector(0)',
```

Here, you probably also don't want `or vector(0)`:

```suggestion
            error_query: 'sum(rate(appcat_probes_seconds_count{reason!="success", service="VSHNPostgreSQL", ha="true"}[{{.window}}]) or 0*rate(appcat_probes_seconds_count{service="VSHNPostgreSQL"}[{{.window}}])) by (service, namespace, name, organization, sla) or sum( 0*rate(appcat_probes_seconds_count{service="VSHNPostgreSQL"}[{{.window}}]) by (service, namespace, name, organization, sla)',
```